src/relay/azext_relay/custom.py

```diff
@@ -0,0 +1,68 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=inconsistent-return-statements
+
+from azext_relay._utils import accessrights_converter
+
+from azext_relay.relay.models import (RelayNamespace, Sku, WcfRelay)
+
+
+# Namespace Region
+def cli_namespace_create(client, resource_group_name, namespace_name, location, tags=None, skutier=None):
+    return client.create_or_update(resource_group_name, namespace_name, RelayNamespace(location, tags, Sku(skutier)))
```

Comment:
You should not rely on positional arguments. It is nearly guaranteed that the SDK will update at some point and break your code. Recommend using kwargs to make your command more resilient to changes in the SDK.

Reply:
implemented